### DISCUSSION TITLE
G7 Not connecting/No QR Code scan possible

### DISCUSSION THREAD

**Discussion by LakeSideCitizen** [NightscoutFoundation/xDrip#2960] [Q&A] [answered]
Good morning all,
meanwhile I'm quite despairring.
Since a few days I'm trying to switch from G6 to G7 with xDrip. 

I followed the instructions **[here](https://navid200.github.io/xDrip/docs/Dexcom/G7.html)**.
After going to **Settings - Auto Configure**  I'm not seeing the message as shown in the instructions.
The message isn't there. 
Instead a camera screen appears directly (see screenshot below).
As it seems it's a scanning screen for the QR-Code but if I try to scan the Settings-QR-Code, or a QR-Code of the G7 sensor nothing happens.
In the Screenhot you can see what happens. The orientation of my scren changes to landscape and there is a bright rectangle, 
what seems to be the scanning section. If i hold a QR-Code from a sensor or the code for xDrip settings in front of it nothing happens.

How can I get the G7 running, respectively connecting to my xDrip?
The Instruction page tells if I can't see a message like **[this](https://navid200.github.io/xDrip/docs/Dexcom/images/keks_QR_confirm.png)** I should stop and inform them.
How can i inform "them" (the developer?) about the issue of the not appearing message as mentioned at the instruction page?

xDrip-Version 23ccbd1-2023.07.16
Samsung Galaxy S20 FE
Androind 13

Can anybody help me?
I don't want to have to use the Dexcom-App.
XDrip is a faithful companion for quite a long time now and I don't wanna miss it.

Thank you very much and have a nice day

regards
Philipp

![Screenshot_20230724_053539_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/39773498/724cd28e-fd72-45bc-81d4-3bb5e188ec8f)

**Comment by Navid200** [comment]
You have added so many screenshots.  They cannot all be relevant, can they?

You say you cannot scan.
Then, why don't you just show us what happens when you tap on "Auto configure"?
If questions come up and in order to answer them, you need to use a screenshot, you can add another.
But, posting a questions with 6 different screenshots, some of which being just copies from our guide, only makes your post unnecessarily cluttered.

You should do what you can to attract people who want to help to your post as opposed to drive them away.

**Reply by LakeSideCitizen** [reply]
The amount of Screenshots ist a mistake because of one Screenshot which wasn't ablebto Upload somehow.
This picture Shows what happens after clicking on "Auto-Configure"
No scanning process ist starting
https://user-images.githubusercontent.com/39773498/255428367-86cae05b-892b-4dc1-99d9-0d7233f7882c.jpg

**Reply by LakeSideCitizen** [reply]
Just want to find a way to start my sensor. Not more!

**Reply by LakeSideCitizen** [reply]
and btw. in this picture you can read "Stop an inform us"
What does it mean exactly? How can I inform them?
https://user-images.githubusercontent.com/39773498/255428364-84e9506c-7a20-446a-a061-78e72d355715.jpg

**Reply by Navid200** [reply]
You can edit the posts and delete the unwanted images.
You can create a single screenshot that shows the entire screen of your phone, not just a portion of it, after you tap on auto config.
I want to help you or find someone who can if you do as I tell you.

**Reply by LakeSideCitizen** [reply]
Done...

**Reply by Navid200** [reply]
Thanks

**Comment by Navid200** [comment]
@jamorham I experience the same thing trying auto configure on a Pixel 6a.

**Reply by jamorham** [reply]
@Navid200 I'm not able to reproduce this on Pixel 6a. There were some relatively recent changes to fix this type of framing issue on larger screens. Those changes should be present in the version the user is using so possibly it is a similar issue that affects this phone type. When this problem existed before it was still possible to scan QR codes just that the viewfinder wasn't displayed correctly on screen.

**Reply by LakeSideCitizen** [reply]
@jamorham, I never tried to Scan a QR-Code before I got my new G7 set, so I can't say wether the problem existed before or not.
I will try to uninstall and re install xDrip after the runtime of my current G6 sensor. Will see

**Comment by Navid200** [comment, marked as answer]
@LakeSideCitizen 
I just uninstalled and reinstalled xDrip and the problem is resolved for me.
But, before doing that, in case you want to give it a try, you need to backup the database.  Otherwise, you will lose all your previous readings in xDrip if you uninstall.

**Reply by LakeSideCitizen** [reply]
I did it as described. Uninstalled and then installed latest xDrip. Everything ist working now.
Thx for support

**Reply by Navid200** [reply]
Fantastic

**Comment by LakeSideCitizen** [comment]
Yes, sure I will give it a try.
The only thing ist, i just placed one of my 2 last G6, because G7 wasn't
running. Remaining time 9 days. Then I will do and yes of course, backups
are running automatically every single day.
Thx for trying to help.

Regards
Philipp


Navid ***@***.***> schrieb am Di., 25. Juli 2023, 15:27:

**Reply by madisondoc7** [reply]
I am having the same problem where the code will not scan and the scanner opens on the side of the screen.
![inbound2220243660598429383](https://github.com/NightscoutFoundation/xDrip/assets/144455722/1fad9d3b-cf51-46a5-9e7c-840e0dad08e6)

**Reply by Navid200** [reply]
Thanks for reporting.   What happens if you restart the phone?

**Comment by madisondoc7** [comment]
Restarting did not fix the problem. Removing xdrip and reinstalling did fix it

**Comment by oakeley** [comment]
Is there a way to kick things when xDrip gets stuck in a "Scanning Errors" loop. My sensor worked fine until yesterday and then just died. It would be nice to be able to restart it rather than discard and add a new one. 
![image](https://github.com/user-attachments/assets/864526ff-0e82-4667-af2e-13bd194bfdaf)

**Reply by Navid200** [reply]
Yes, of course
But, that is not related to the specific subject of this thread.  
Please start a new thread with a relevant title.

**Comment by Navid200** [comment]
Starting with the August 2, 2024 release, there is an option to scan a QR code from an image file saved in storage.
